<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigatorWidget</class>
 <widget class="QDockWidget" name="NavigatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>712</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>524287</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>440</width>
    <height>665</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Selection / Filter</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>418</width>
      <height>650</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>435</width>
      <height>650</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>419</width>
       <height>664</height>
      </rect>
     </property>
     <widget class="QComboBox" name="working_l1_cbox">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>4</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="working_l2_cbox">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>4</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="error_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>576</y>
        <width>374</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {color : red;}
font: 75 14pt &quot;Helvetica&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="load_exc_button">
      <property name="geometry">
       <rect>
        <x>316</x>
        <y>67</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>EXCEL</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="exc_twidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>381</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="exc_path_edit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="header_row_sbox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="data_row_sbox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="data_column_sbox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Header row</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>data row</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>data column</string>
      </property>
     </widget>
     <widget class="QPushButton" name="load_data_button">
      <property name="geometry">
       <rect>
        <x>316</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>LOAD</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
